test/routing-tables.test.js

```diff
@@ -547,114 +562,6 @@ describe('RoutingTables', function () {
       }, /TypeError: RoutingTables#toJSON maxPoints must be a positive number/)
     })
   })
-
-  describe('findBestHopForDestinationAmount', function () {
-    it('finds the best route when there is one path', function () {
```

Comment:
why was this test removed? there still is a findBestHopForDestinationAmount function, right?